includes/API.php

```diff
@@ -62,12 +66,20 @@ public function send_item_updates( $requests, $allow_upsert ) {
 	 *
 	 * @since 2.0.0-dev.1
 	 *
-	 * @param string $catalog_id
-	 * @param array $data
+	 * @param string $catalog_id catalog ID
+	 * @param array $data product group data
+	 * @return Response
+	 * @throws Framework\SV_WC_API_Exception
 	 */
 	public function create_product_group( $catalog_id, $data ) {
 
-		// TODO: Implement create_product_group() method.
+		$request = $this->get_new_request( [ $catalog_id, '/product_groups', 'POST' ] );
```

Comment:
I think the approach you mentioned earlier, to have the `API\Request` class receive the path and let users of the class build it however they need is much better. I'll create a story to make that update. Thank you for bringing that up!